rosidl_parser/rosidl_parser/definition.py

```diff
@@ -595,6 +596,20 @@ def __init__(
             namespaced_type.name + SERVICE_RESPONSE_MESSAGE_SUFFIX
         self.response_message = response
 
+        self.event_message = Message(
```

Comment:
shouldn't the event message be passed as an argument?
similar to request and response